src/labels/CustomLabels.labels

```diff
@@ -2403,6 +2419,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <value>Account Data</value>
     </labels>
     <labels>
+        <fullName>healthLabelAccountDefaultRT</fullName>
+        <categories>Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>healthLabelAccountDefaultRT</shortDescription>
+        <value>Account Model Record Type</value>
```

Comment:
I'm not sure about this title for the check. @smarovich Would you think about "Account Model -- Selected Record Type" instead?

Reply:
I suggested this title and would like to stick with it.